### ISSUE TITLE
Crash at "generating deterministic UUIDs" when running pod install

### ISSUE THREAD

**Issue by jefflovejapan** [CocoaPods/CocoaPods#6290]
## What did you do?

Run `pod install`

## What did you expect to happen?

I expected CocoaPods to download my project's dependencies, generating targets for each and a workspace containing all of the targets.

## What happened instead?

CocoaPods crashed, seemingly on "Generating deterministic UUIDs". I've tried uninstalling / reinstalling CocoaPods, `pod cache clean --all`, `pod deintegrate`, etc. and still get the same crash.

```
Processing target: gciosNetworking...
Processing target: Pods-GC-GC Bats...
Processing target: Pods-GC-GCTests...
    - Podfile
  - Writing Xcode project file to `Pods/Pods.xcodeproj`
    - Generating deterministic UUIDs
[1]    11969 abort      pod install --verbose
```

### Stack

```
   CocoaPods : 1.1.1
        Ruby : ruby 2.2.3p173 (2015-08-18 revision 51636) [x86_64-darwin14]
    RubyGems : 2.4.5.1
        Host : Mac OS X 10.12.1 (16B2659)
       Xcode : 8.1 (8B62)
         Git : git version 2.6.2
Ruby lib dir : /Users/jeff/.rvm/rubies/ruby-2.2.3/lib
Repositories : gamechanger - git@github.com:gamechanger/gcpodspecs.git @ 48dc03d4abe1ef02dc1df2217b2d12162649e6bc
               master - https://github.com/CocoaPods/Specs.git @ e0e4852dd8d3bc778746d4c97b0397318bf3ee54
```

### Installation Source

```
Executable Path: /Users/jeff/.rvm/gems/ruby-2.2.3@gcmobile/bin/pod
```

### Plugins

```
cocoapods-deintegrate : 1.0.1
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-stats       : 1.0.0
cocoapods-trunk       : 1.1.1
cocoapods-try         : 1.1.0
```

### Podfile

```ruby
platform :ios, "9.0"
workspace 'gcios_batsi'
project 'baseball_sc/GameChanger.xcodeproj'
source 'git@github.com:gamechanger/gcpodspecs.git'
source 'git@github.com:CocoaPods/Specs.git'
inhibit_all_warnings!
use_frameworks!

abstract_target 'GC' do
  pod 'AWSS3', '2.4.1'
  pod 'CocoaLumberjack', '2.3.0'
  pod 'DeepLinkKit', '1.0.0'
  pod 'ECPhoneNumberFormatter', '0.1.1'
  pod 'FMDB', '2.5'
  pod 'GCGRMustache', '1.6.1-gc.4'
  pod 'GCPartnerDBs', '~> 1.3'
  pod 'GCMAggregatingTableViewDataSource', '0.1.8'
  pod 'GCAppirater', '3.0.7'
  pod 'GCMCocoaExtensions', '0.2.7'
  pod 'GCMFormTableView', '0.6.14'
  pod 'GCStatEngine', '0.7.1'
  pod 'GCUIKit', '2.0'
  pod 'GCYBStatechart', '1.0.3'
  pod 'Godzippa', '1.1.0'
  pod 'GoogleAnalytics', '3.14.0'
  pod 'HockeySDK', '4.0.1'
  pod 'MMDrawerController', '0.5.3'
  pod 'PromiseKit/CorePromise', '~> 4.0'
  pod 'Reachability', '3.2'
  pod 'Rollbar', '0.2'
  pod 'UIDeviceIdentifier', '0.4.0'
  pod 'UICKeyChainStore', '2.1.0'
  pod 'FBSDKCoreKit', '4.11.0'
  pod 'SnowplowTracker', '0.6.1'
  pod 'Sabertooth', '4.0.3'
  pod 'gciosNetworking', '2.1.0'
  pod 'Firebase', '3.8.0'
  pod 'FirebaseMessaging', '1.2.0'

  pod 'AppFriendsUI', :git => 'git@github.com:Hacknocraft/AppFriendsUI.git', :tag => '1.0.32'
  pod 'AppFriendsCore', :git => 'git@github.com:Hacknocraft/AppFriendsCore.git', :tag => '0.1.77'
  pod 'CoreStore', :git => 'git@github.com:JohnEstropia/CoreStore.git', :commit => '92756fec42a925bc5c0845571bbf575e414f9ab9'

  target 'GC Bats' do
  end

  target 'GCTests' do
    #pod 'Kiwi', '2.4.0'#, :inhibit_warnings => true
    pod 'Kiwi', :git => "https://github.com/kiwi-bdd/Kiwi.git", :commit => '1d83dc1e0e98b73eb1debe02cdb345ae230b1f75'
    pod 'OCMock', '3.3'#, :inhibit_warnings => true
  end
end

post_install do | installer |
  require 'fileutils'
  FileUtils.cp_r('Pods/Target Support Files/Pods-GC-GC Bats/Pods-GC-GC Bats-acknowledgements.plist', 'baseball_sc/Common/Resources/Settings.bundle/Acknowledgements.plist', :remove_destination => true)
  installer.pods_project.targets.each do |target|
    puts "Processing target: #{target.name}..."
    target.build_configurations.each do |config|
      config.build_settings['ARCHS'] = "$(ARCHS_STANDARD)"
      config.build_settings['CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES'] = 'YES'
      config.build_settings['ENABLE_BITCODE'] = 'NO'
      config.build_settings['SWIFT_VERSION'] = "3.0.1"
      config.build_settings['EXPANDED_CODE_SIGN_IDENTITY'] = ""
      config.build_settings['CODE_SIGNING_REQUIRED'] = "NO"
      config.build_settings['CODE_SIGNING_ALLOWED'] = "NO"
    end
  end
end
```

**Comment by jefflovejapan** [comment]
This appears to be fixed in 1.2.0.beta.1, closing.

**Comment by mrousavy** [comment]
I think I'm experiencing a similar issue [here](https://github.com/CocoaPods/CocoaPods/issues/10506), that's on CocoaPods 1.10.1 :(
